lib/tesla/adapter/mint.ex

```diff
@@ -1,41 +1,47 @@
 if Code.ensure_loaded?(Mint.HTTP) do
   defmodule Tesla.Adapter.Mint do
     @moduledoc """
-      Adapter for [mint](https://github.com/ericmj/mint)
-
-      Caution: The minimum supported Elixir version for mint is 1.5.0
-
-      Remember to add `{:mint, "~> 0.2.0"}` and `{:castore, "~> 0.1.0"}` to dependencies
-      Also, you need to recompile tesla after adding `:mint` dependency:
-
-      ```
-      mix deps.clean tesla
-      mix deps.compile tesla
-      ```
-
-      ### Example usage
-      ```
-      # set globally in config/config.exs
-      config :tesla, :adapter, Tesla.Adapter.Mint
-
-      # set per module
-      defmodule MyClient do
-        use Tesla
-
-        adapter Tesla.Adapter.Mint
-      end
-
-      # set global custom cacert
-      config :tesla, Tesla.Adapter.Mint, cacert: ["path_to_cacert"]
+    Adapter for [mint](https://github.com/ericmj/mint)
```

Comment:
The repo is moved to https://github.com/elixir-mint/mint

Reply:
👍